examples/trips/README.md

```diff
@@ -0,0 +1,2 @@
+* [hello-world-webpack2](./hello-world-webpack.md): Bundles a minimal app with
```

Comment:
Copy paste... Needs content.

Also the README in the parent examples folder should be updated.